### DISCUSSION TITLE
More types in GrapqhQL API

### DISCUSSION THREAD

**Discussion by belafonte** [Cockpit-HQ/Cockpit#97] [General]
Hey there,

Currently there a some types not handled by GraphQL but instead a generic JsonType is returned.
For instance for Assets.
I saw there is a "Upload Type" but it seems unused ?

For codegen and typechecking it would be nice to have types for e.g. images/assets since they seem to be defined somewhere anyway.

**Comment by aheinze** [comment]
Thanks for bringing this up. I'll look into it. 👍
